ocaml/xapi/xapi_vbd_helpers.ml

```diff
@@ -169,12 +169,12 @@ let valid_operations ~expensive_sharing_checks ~__context record _ref' : table =
     if not vdi_record.Db_actions.vDI_managed
     then set_errors Api_errors.vdi_not_managed [ _ref ] all_ops;
 
-    let vdi_operations_besides_copy =
+    let operation_cannot_be_performed_live=
       List.exists
-        (fun (_, operation) -> operation <> `copy)
```

Comment:
Looks like the complete list can be read in `check_operation_error` and `check_sm_feature_error` in `xapi_vdi.ml`